Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -48,6 +48,10 @@ extension BrowserViewController {
 
             return Self.allCases[safe: indexOfSelf + 1]
         }
+        
+        var value: Int {
+            return AppConstants.buildChannel.isPublic ? self.rawValue : self.rawValue / 10
```

Comment:
pushing that one line `return` keyword!! :D

Reply:
See I didnt do that on purpose it happened :)